### PR TITLE
fixing pagination for medialibrary

### PR DESCRIPTION
**Pull request by hbaptiste**
fixing pagination for medialibrary

### CODE REVIEW

File: Rest/Controller/ClassContentController.php [hunk 1]
```diff
@@ -114,7 +117,15 @@ public function getCollectionAction($start, $count, Request $request)
...
 
-        return $this->addContentRangeHeadersToResponse($response, $contents, $start);
+        if (!$usePagination) {
+            if ( $contents instanceof Paginator) {
+                $resultCount = count($contents->getIterator());
+            } else {
+                $resultCount = count($contents);
+            }
+        }
+
```
Comment (crouillon) [inline Rest/Controller/ClassContentController.php]: This block seems to be useless

File: Rest/Controller/ClassContentController.php [hunk 2]
```diff
@@ -658,7 +669,12 @@ private function findContentsByCriteria(array $classnames, $start, $count)
             ->findContentsBySearch($classnames, $order_infos, $pagination, $criterias)
         ;
 
-        foreach ($contents as $content) {
+        $contentsItem = $contents;
+        if ($contents instanceof Paginator) {
+            $contentsItem = $contents->getIterator();
+        }
+
```
Comment (crouillon) [inline Rest/Controller/ClassContentController.php]: Paginator implements Traversable, so it could be directly iterate by foreach

**Comment by crouillon** [comment]
:+1:

**Comment by hbaptiste** [comment]
:+1:

**Pull request merged**